test/test-config.sh

```diff
@@ -86,6 +86,10 @@ fi
 # This string is used as part of deployment file name.
 : ${TEST_DEVICEMODE:=lvm}
 
+# Which deployment to use during testing.
+# Allowed values: testing (default), production
+: ${TEST_DEPLOYMENTMODE:=testing}
```

Comment:
Following the name formatting mostly used in `test/test-config.sh` file, this variable should be `TEST_DEPLOYMENT_MODE`.
 Although if we decide to change it, we should change the name of `TEST_DEVICE_MODE` too, but we can track this specific change in a different PR, what do you think?

Reply:
After reading a bit more the `README.md` file, `DEVICEMODE` is used as one word, so the name `TEST_DEVICEMODE` follows the name formatting in `test/test-config.sh`.

Is `DEPLOYMENTMODE` ging to be used in the same way? If yes, then you can ignore my comment to rename it as `TEST_DEPLOYMENT_MODE`. If no, then we need to rename it.

Reply:
I was struggling with that "to underscore or not to underscore" question myself. I settled on making it the same as `DEVICEMODE` simply for the sake of consistency, even though strictly speaking with underscore would make more sense.

Reply:
Got it!